salt/gobblin/init.sls

```diff
@@ -83,6 +83,7 @@ gobblin-install_gobblin_pnda_job_file:
 gobblin-create_gobblin_logs_directory:
   file.directory:
     - name: /var/log/pnda/gobblin
+    - user: pnda
```

Comment:
the user value is currently hardcoded, could you please use this instead:
{{ pnda_user }}
as this is the value get from the pillars.

Reply:
Made the above change..
Thanks for the feedback!